### PR TITLE
DOCS-3268: Update Currencies

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/my-account/currencies.md [hunk 1]
```diff
@@ -6,21 +6,39 @@ hidden: false
 parentDoc: 62b0845857c8ab006af6a4f7
 slug: 'currencies'
 ---
+
+You can process payments in a large number of currencies. When customers can pay in their preferred local currency, it increases their trust in your site and therefore your <<glossary:conversion rate>>.
+
 # How it works
 
-You can process payments in a large number of currencies, provided that they are:
+To accept payments in a currency other the default (EUR), the currency must be:
```
Comment (Sarah-MultiSafepay) [inline content/my-account/currencies.md:14]:
```suggestion
To accept payments in a currency other than the default (EUR), the currency must be:
```

File: content/my-account/currencies.md [hunk 2]
```diff
@@ -42,9 +108,9 @@ Once your new business bank account has been approved by the Risk Team, you can
 
 To convert other currencies to Euros (EUR) in your [MultiSafepay dashboard](https://merchant.multisafepay.com), go to **Finance > Currency conversion**. 
 
-Take into account the exchange rate and/or any other costs.
+Take into account the exchange rate and MultiSafepay's 4% conversion fee.
```
Comment (Sarah-MultiSafepay) [inline content/my-account/currencies.md:111]: Can you add a callout to make this stand out a bit more?

File: content/my-account/currencies.md [hunk 3]
```diff
@@ -6,21 +6,39 @@ hidden: false
...
-- Supported by the specific payment method
-- Enabled in your dashboard and in the settings of any [ready-made integrations](/docs/our-integrations/)
-- Correctly processed in the [create order](/reference/createorder/) API request
+- Supported by the payment method
+- Enabled for your account 
+- Supported in your <<glossary:backend>>
+- Correctly processed in the [create order](/reference/createorder/) request
 
-# Activation
+You can support up to 12 currencies per account ID.
```
Comment (Sarah-MultiSafepay) [inline content/my-account/currencies.md:21]: Can you enable/show different currencies per site ID?

Comment (Sarah-MultiSafepay) [reply content/my-account/currencies.md:21]: Can we add instructions for that use case? I think that would be useful for merchants.

Comment (FrancescaNCook) [reply content/my-account/currencies.md]: @Sarah-MultiSafepay Turns out the answer is no you can't! I've removed this bit.

**Review by Sarah-MultiSafepay — APPROVED** [review]

**Pull request merged**